### ISSUE TITLE
Support for GHC 8.10.1

### ISSUE THREAD

**Issue by InBetweenNames** [tweag/asterius#521]
Hi Asterius Team,

Are there any plans to port Asterius over to GHC 8.10.1?

**Comment by TerrorJack** [comment]
Yes. We plan to do that when another major version of stackage lts snapshot with ghc-8.10 is released.

**Comment by InBetweenNames** [comment]
Wonderful!  Keep up the good work!
